src/components/Analysis/HistoryButtons/HistoryButtons.tsx

```diff
@@ -53,7 +54,7 @@ export class HistoryButtons extends React.Component<HistoryButtonsProps> {
     } = this.props;
 
     return (
-      <div>
+      <>
```

Comment:
Use '<Fragment>' instead of <>. Its basically the same but it makes clearer that you really want that

Reply:
Not sure that works?
https://reactjs.org/docs/fragments.html

Reply:
There should be a Fragmet component but its hidden somehow. Strange

Reply:
What do you mean?

Reply:
yes. I just think using the actual Component Fragment makes the code clearer

Reply:
Don't agree here. If you use <> it's clearer that there isn't a DOM element used here.
But that kind of stuff is more a personal preference.